### PR TITLE
GSYE-362: Renamed the historical data endpoint documentation from his…

### PR DESCRIPTION
**Pull request by spyrostz**
…torical-data/devices to historical-data/assets in order to reflect the updated endpoint.

## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

**Review by mrsaemir — COMMENTED** [review]
Just a few points. LGTM in general.

File: wiki/docs/get-simulation-data.md [hunk 1]
```diff
@@ -77,28 +77,26 @@ username = "<your_user>"
...
-response = requests.post(url=domain + "/api-token-auth/", 
+response = requests.post(url=domain + "/api-token-auth/",
                          json={"username": username, "password": password})
 
 token = response.json()["token"]
 ```
 3. Create a URL and send the request:
 ```python
-historical_data_url = f"{domain}/historical-data/devices/{configuration_id}/{device_id}/{start_date}/{end_date}"
+historical_data_url = f"{domain}/historical-data/assets/{configuration_id}/{device_id}/{start_date}/{end_date}"
```
[the remark this answers is not in the document]
Comment (spyrostz) [reply wiki/docs/get-simulation-data.md:94]: It should be yes, I am not sure if the customer scripts that the research team has created should correspond to this, however it is easy to figure out for the reader. Changing it, thanks

File: wiki/docs/releases.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ The following features were implemented :
 
 * [Grid fees](grid-fees.md) configuration
 * User roles, including 'Collaborations' feature for multi user testing and Grid Singularity-hosted challenges
-* [API](configure-trading-strategies-walkthrough.md) connection to energy devices allowing bids and offers to be submitted by external algorithms
+* [API](configure-trading-strategies-walkthrough.md) connection to energy assets allowing bids and offers to be submitted by external algorithms
```
[the remark this answers is not in the document]
Comment (spyrostz) [reply wiki/docs/releases.md:133]: Fixing, thanks

**Review by cruky — APPROVED** [review]
LGTM

**Review by mrsaemir — APPROVED** [review]
LGTM

**Pull request merged**